app/cdap/components/PipelineList/DeployedPipelineView/PipelineTable/index.tsx

```diff
@@ -28,27 +27,16 @@ import './PipelineTable.scss';
 
 interface IProps {
   pipelines: IPipeline[];
-  filteredPipelines: IPipeline[];
   search: string;
   onClear: () => void;
   refetch: () => void;
 }
 
 const PREFIX = 'features.PipelineList';
 
-const PipelineTableView: React.SFC<IProps> = ({
-  pipelines,
-  filteredPipelines: filteredList,
-  search,
-  onClear,
-  refetch,
-}) => {
+const PipelineTableView: React.SFC<IProps> = ({ pipelines, search, onClear, refetch }) => {
```

Comment:
i know you just removed a prop but in the future can you remove the : React.SFC<> in favor of `({}: IProps) =>`